normandy/settings.py

```diff
@@ -212,6 +212,7 @@ def AUTHENTICATION_BACKENDS(self):
 
     OIDC_REMOTE_AUTH_HEADER = values.Value('HTTP_REMOTE_USER')
     OIDC_LOGOUT_URL = values.Value(None)
+    OIDC_USER_ENDPOINT = values.URLValue('https://auth.mozilla.auth0.com/userinfo')
```

Comment:
We may want to either distinguish this new auth method from the previous "OIDC" settings/middlewares here. Having a setting `USE_OIDC` that isn't actually related to the OIDC middle ware sounds confusing.

I don't think we can completely remove it yet though. It may be good to have two separate config presets though.

Reply:
Naming things are hard. 

I like you point that it is confusing that *this* OIDC stuff we're doing now is muddled with the old OIDC stuff. And we can't delete all the old stuff yet, can we?

One nice start would be to create a new class, similar to [CORS](https://github.com/peterbe/normandy/blob/b82b74618cfde4b7ee014d213f45cbbc7cfa1958/normandy/settings.py#L176) but dedicated to this OIDC-authentication-by-access_token. Then it's at least not near the "old OIDC" stuff. 
Ideas for a name (of the class)?